maintainers/team-list.nix

```diff
@@ -1091,9 +1091,10 @@ with lib.maintainers;
 
   vim = {
     members = [
-      figsoda
-      softinio
-      teto
+      GaetanLepage
+      khaneliman
+      mrcjkb
+      perchun
     ];
```

Comment:
We can specify `githubTeams = [ "neovim" ];` instead of listing all members by hand



https://github.com/NixOS/nixpkgs/blob/a3e92c317832e85291a3eb4006dbd1c9f757ea74/maintainers/team-list.nix#L682-L687